Makefile

```diff
@@ -1,10 +0,0 @@
-
```

Comment:
Is the Makefile used anywhere except Travis CI ?  
There were two sections in Makefile out of which the second one was never used, because _JOBLIB\_MULTIPROCESSING_ environemnt variable is set at appropriate job in config file. And the first section only had one command so I simply transferred it in the script. If you like the original setup then sure I'll restore it back.